conf/pf.conf.defaults

```diff
@@ -817,6 +817,17 @@ secure_redirect=enabled
 # If status_only_on_production is enabled, the /status page will only be available on the
 # production network. This is disabled by default
 status_only_on_production=disabled
+#
+# captive_portal.detection_mecanism_urls
+#
+# Comma-delimited list of URLs known to be used by devices to detect the presence 
+# of a captive portal and trigger their captive portal mecanism.
+detection_mecanism_urls = http://www.gstatic.com/generate_204,http://clients3.google.com/generate_204,http://www.apple.com/library/test/success
```

Comment:
Don't we have to detect all the apple domain ?
Or maybe can we use the user-agent ? User-Agent: CaptiveNetworkSupport/1.0 wispr